Farmacy/Models/Conversation.m

```diff
@@ -0,0 +1,17 @@
+//
+//  Conversation.m
+//  Farmacy
+//
+//  Created by Trang Dang on 8/1/22.
+//
+
+#import "Conversation.h"
+
+@implementation Conversation
+@dynamic userOne;
+@dynamic userTwo;
```

Comment:
I'm not familiar with the syntax of `@dynamic` in objc, but I thought these were already defined as properties in Conversation.h?